### PR TITLE
Add initial service support and datastream content filtering

### PR DESCRIPTION
**Pull request by frwickst**
I suggest going through the commits one by one and asking about the things that are unclear (I guess there are some 😄).

### CODE REVIEW

File: backend/gatekeeper/views.py [hunk 1]
```diff
@@ -186,8 +159,227 @@ def create(self, request, sts_response):
...
+        expand_params = params.get('$expand', '')
+
+        # If we are already extending Datastreams there is no need for further actions
+        if expand_params and self.sts_datastreams_extend_re.match(expand_params):
+            return False
+
+        parse_result = parse_sta_url(self.sts_url, prefix=get_gatekeeper_sta_prefix())
+
+        entity_type = parse_result['parts'][-1]['name']
+        print("Entity type: ", entity_type)
```
Comment (mikkokeskinen) [inline backend/gatekeeper/views.py]: Print debug

Comment (frwickst) [reply backend/gatekeeper/views.py]: Good catch, did not know that flake8-print wasn't added to the project so I added that as well 😄

File: backend/gatekeeper/views.py [hunk 2]
```diff
@@ -215,7 +392,15 @@ def handle_request(self, request):
 
             # Pop the content type as we want to allow both JSON and browsable API responses
             response_args.pop('content_type')
-            response_data = exclude_unauthorized_data(sts_response.json(), request.user)
+
+            response_data = json_content
+            try:
+                if request.method == 'GET':
+                    pass
```
Comment (mikkokeskinen) [inline backend/gatekeeper/views.py]: unnecessary pass here

**Comment by suutari-ai** [comment]
Good work guys! :+1: Let's merge this! :smile_cat:

**Pull request merged**